src/yacfg/yacfg.py

```diff
@@ -136,6 +136,11 @@ def override_value_map_keys(value: Dict[str, Any]) -> Dict[str, Any]:
         """
         return {override_value(key, key): val for key, val in value.items()}
 
+    def zip_lists(value: List[Any], pairing: List[Any]) -> List[(Any, Any)]:
```

Comment:
Isn't this the invalid syntax for tuple types we discussed last week?

Reply:
I don't think so. It currently runs for me. Will check to make sure and resolve this.

Reply:
```
>>> from typing import List, Any
>>> List[(Any, Any)]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.12/typing.py", line 384, in inner
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/typing.py", line 1436, in __getitem__
    _check_generic(self, params, self._nparams)
  File "/usr/lib64/python3.12/typing.py", line 304, in _check_generic
    raise TypeError(f"Too {'many' if alen > elen else 'few'} arguments for {cls};"
TypeError: Too many arguments for typing.List; actual 2, expected 1
```